sktime/forecasting/compose/_hierarchy_ensemble.py

```diff
@@ -109,10 +143,20 @@ class HierarchyEnsembleForecaster(_HeterogenousEnsembleForecaster):
 
     _steps_attr = "_forecasters"
 
-    def __init__(self, forecasters, by="level", default=None):
+    def __init__(
+        self, forecasters, by="level", default=None, backend=None, backend_params=None
+    ):
         self.forecasters = forecasters
         self.by = by
         self.default = default
+        self.backend = backend
+        self.backend_params = backend_params
+        self.set_config(
```

Comment:
I would remove the set_config from this - these will be inactive because the estimator does not broadcas.t